lib/__tests__/InstantSearch-test.js

```diff
@@ -176,4 +176,56 @@ describe('InstantSearch lifecycle', () => {
       expect(algoliasearchHelper.args[0][2].values).toEqual([-2, -1, 0, 1, 3, 4, 2]);
     });
   });
+
+  context('when render happens', () => {
+    var renderCalled;
+    beforeEach(() => {
+      renderCalled = 0;
```

Comment:
What about using sinon.js for spies like the previous tests are using? Of course the current spy code
you did is really simple and clean but let's use the tools we have even if there's a learning curve?

```js
var renderCalled = sinon.spy()
// ...
return {
  render: renderCalled
}

expect(renderCalled.callCount).toEqual(0);
```

Same thing for renderEmitted.